frontend/components/dashboard/OverviewBox.vue

```diff
@@ -22,7 +22,8 @@ const openValidatorModal = () => {
     data: {
       context: 'dashboard',
       dashboardName: getDashboardLabel(dashboardKey.value, 'validator'),
-      dashboardKey: dashboardKey.value
+      dashboardKey: dashboardKey.value,
+      timeFrame: 'last_24h'
```

Comment:
@LuccaBitfly could the backend be adapted not to require a `period parameter`

![image](https://github.com/user-attachments/assets/9a0fad28-2b31-4625-aeea-fde6b42332ba)
![image](https://github.com/user-attachments/assets/864e370e-6346-4d40-910a-13620d397ce3)

Reply:
@marcel-bitfly it used to be like that, but at some point someone requested that the API should return an error if this parameter is omitted. Can revert that again if you want to. pls lmk

Reply:
@MauserBitfly suggested to leave it like that, as we might break other things.  Thx for the information